lib/puppet/type/gluster_volume.rb

```diff
@@ -0,0 +1,62 @@
+Puppet::Type.newtype(:gluster_volume) do
+  desc <<-'ENDOFDESC'
+  Defines a glusterfs peer to probe.
+
+    Example:
+
+      gluster_volume { 'volume1':
+        replica => 2,
```

Comment:
`replica` is the command line option, so in the absence of any compelling reasons either way I stuck with that rather than the plural. I don't have strong opinions either way.